engines/director/director.h

```diff
@@ -174,6 +176,12 @@ class DirectorEngine : public ::Engine {
 	Movie *getCurrentMovie() const;
 	void setCurrentMovie(Movie *movie);
 	Common::String getCurrentPath() const;
+	Common::String getCurrentAbsolutePath() const { 
+		return Common::String::format("%s%s", 
+				getPlatform() == Common::kPlatformWindows ? "C:\\" : "",
+				getCurrentPath().c_str()
```

Comment:
Do you have the right path separators here? E.g. may it lead to `"C:\Director/game/directory"`?

Reply:
Good catch, should be fixed now